src/hooks/calculateTodo.js

```diff
@@ -0,0 +1,11 @@
+export const getTotalTodos = (todoLists) => {
+  return todoLists.length;
+};
+
+export const getCheckedTodos = (todoLists) => {
+  return todoLists.filter((todoList) => todoList.checked).length;
+};
+
+export const getProceedingTodos = (todoLists) => {
+  return getTotalTodos(todoLists) - getCheckedTodos(todoLists);
+};
```

Comment:
사람마다 다르지만 하나의 return 만 있을때에는 중괄호와 return 키워드를 생략할 수도 있습니다!
```suggestion
export const getTotalTodos = (todoLists) => todoLists.length;

export const getCheckedTodos = (todoLists) => todoLists.filter((todoList) => todoList.checked).length;

export const getProceedingTodos = (todoLists) => getTotalTodos(todoLists) - getCheckedTodos(todoLists);
```

Reply:
피드백 감사합니다😄